packages/terra-application-layout/src/terra-dev-site/doc/common/ApplicationMenu.jsx

```diff
@@ -59,7 +63,7 @@ class ApplicationMenu extends React.Component {
                   layoutConfig.toggleMenu();
                 }
               }}
-              style={{ marginBottom: '5px' }}
+              className={cx('application-layout-common-application-menu-custom-event')}
```

Comment:
`event`? I'd say it's a `button`.

Reply:
Changed the class name to be more appropriate and short, i.e. 'custom-event-button'.